src/components/tabs/tabs.js

```diff
@@ -218,10 +233,25 @@ Tabs.prototype.activateNextTab = function () {
 }
 
 Tabs.prototype.activatePreviousTab = function () {
+  // IE doesn't support 'previousElementSibling' this code polyfills IE8
```

Comment:
If we do go down the polyfill route, this is how we tend to do it: https://github.com/alphagov/govuk-frontend/blob/master/docs/contributing/polyfilling.md